zer0-os/zOS
# Check for isDistinct when deteriming if a conversation is one on one

### What does this do?

Changes how we decide a conversation is a one on one. A convo is one on one IF:
1. The conversation `isDistinct` is true and
2. there is exactly 1 other member

### Why are we making this change?

A group conversation can have enough members leave that it ends up having only 1 other member. In this case we were previously rendering it as a one on one conversation which hides author names and avatars making it impossible to know who sent previous messages.

### How do I test this?

Create a group conversation with 3 people. Send messages from each of those people. Remove one from the conversation. Verify the convo still renders as a group conversation for the remaining members.
